src/views/onboarding/__tests__/Onboarding.test.tsx

```diff
@@ -333,6 +333,13 @@ const fillUserBillingDataForm = async (
   fireEvent.change(document.getElementById(taxCodeInput), {

```

Comment:
let's add to completeTest also a check to verify if the submit has been performed with the exected information